<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;SansSerif&quot;;
color: black;</string>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>518</x>
      <y>370</y>
      <width>151</width>
      <height>129</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 221, 0);
border-bottom-right-radius: 40px;
border-top-left-radius: 121px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="discard">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>440</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;SansSerif&quot;;
color: black;
text-align: center;
background-color:rgba(0, 0, 0, 0.1);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="back_g">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>561</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
background-color: rgb(255, 255, 255);
border-radius: 40px;	</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="fs_text">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgba(2, 2, 2, 0.1);
font: 10pt &quot;SansSerif&quot;;
color: rgb(0,0,0)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Enter the value of Fs...</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>440</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;SansSerif&quot;;
color: black;</string>
    </property>
    <property name="text">
     <string>IIR Butterworth Filter:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>62</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 221, 0);
border-top-left-radius: 40px;
border-bottom-right-radius: 121px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="n_text">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgba(2, 2, 2, 0.1);
font: 10pt &quot;SansSerif&quot;;
color: rgb(0,0,0)</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Enter the value of N...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Show_Graph">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>440</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;SansSerif&quot;;
color: black;
text-align: center;
background-color:rgba(0, 0, 0, 0.1);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>Show Graph</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;SansSerif&quot;;
color: black;</string>
    </property>
    <property name="text">
     <string>Fs</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;SansSerif&quot;;
color: black;</string>
    </property>
    <property name="text">
     <string>Fc</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fc_text">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgba(2, 2, 2, 0.1);
font: 10pt &quot;SansSerif&quot;;
color: rgb(0,0,0)</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Enter the value of Fc...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>back_g</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_2</zorder>
   <zorder>label_7</zorder>
   <zorder>Show_Graph</zorder>
   <zorder>n_text</zorder>
   <zorder>fs_text</zorder>
   <zorder>label_3</zorder>
   <zorder>label_6</zorder>
   <zorder>label_8</zorder>
   <zorder>discard</zorder>
   <zorder>label_11</zorder>
   <zorder>label_12</zorder>
   <zorder>label_9</zorder>
   <zorder>fc_text</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
